doc/nrf/gs_modifying.rst

```diff
@@ -58,7 +58,7 @@ In the window that is displayed, you can define compilation options for the proj
 
 .. note::
    These compilation options apply to the application project only.
-   To manage Zephyr and other subsystems, go to :guilabel:`Project` > :guilabel:`Configure nRF Connect SDK Project`.
+   To manage Zephyr and other subsystems, go to :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK Project`.
```

Comment:
That's different than what DITA does. I'm okay with it if that's how we want to do it, but it needs to be defined.

Reply:
Yes, just wanted to have consistency here, as there were different options. We can discuss it at one of the meetings after 1.4.0.